tests/does_not_exist.html

```diff
@@ -0,0 +1 @@
+temp
```

Comment:
What is the reason for that file?

Reply:
I was experimenting with the mozdownload's codebase mistakenly added. Will take care of it.